src/App.tsx

```diff
@@ -1,156 +1,82 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  ChangeEvent,
+  useEffect,
+  useState,
+} from 'react';
+
+import { getUsers } from './api/users';
+import { getTodos } from './api/todos';
+import { Login } from './Login';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Filter } from './components/Filter/Filter';
+import { NewTodo } from './components/NewTodo/NewTodo';
+import NotificationError from
+  './components/NotificationError/NotificationError';
 
-const USER_ID = 0;
+const USER_ID = localStorage.getItem('userId');
 
 export const App: React.FC = () => {
+  const [task, setTask] = useState('');
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+
+  const [error, setError] = useState<string>('');
+
+  const handleTodoChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setTask(event.target.value);
+  };
+
+  const loadTodos = async () => {
+    try {
+      await getTodos(Number(USER_ID)).then(res => setTodos(res));
+    } catch {
+      setError('unable to get todos');
+    }
+  };
+
+  useEffect(() => {
+    getUsers();
+  }, [USER_ID]);
+
+  useEffect(() => {
+    if (USER_ID) {
+      loadTodos();
+    }
```

Comment:
The USER_ID will never change. Look how it works now 
<img width="546" alt="image" src="https://user-images.githubusercontent.com/99403198/230579957-990deb67-0a3d-41ba-b3c7-12114f97c7a2.png">
It's just an infinite loader here 

https://user-images.githubusercontent.com/99403198/230580098-6dc6ccc8-f100-449c-8df0-33e8729bb90e.mov